deployment/network-operator/README.md

```diff
@@ -122,6 +123,55 @@ resources:
 >__Note__: The parameter listed are non-exahustive, for the full list of chart parameters refer to
 the file: `values.yaml`
 
+#### Secondary Network
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `secondaryNetwork.deploy` | bool | `true` | Deploy Secondary Network  |
+
+Specifies components to deploy in order to facilitate a secondary network in Kubernetes. It consists of the folowing optionally deployed components:
+  - [Multus-CNI](https://github.com/intel/multus-cni)
+  - CNI plugin: Currently only [Macvlan CNI](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan) is supported
+  - IPAM CNI: Currently only [Whereabout IPAM CNI](https://github.com/openshift/whereabouts-cni) is supported
+
+##### CNI Plugin Secondary Network
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `cniPlugin.deploy` | bool | `false` | Deploy CNI Plugin Secondary Network  |
+| `cniPlugin.image` | string | `None` | CNI Plugin image name  |
+| `cniPlugin.repository` | string | `None` | CNI Plugin image repository  |
+| `cniPlugin.version` | string | `None` | CNI Plugin image version  |
+
+##### Multus CNI Secondary Network
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `multus.deploy` | bool | `true` | Deploy Multus Secondary Network  |
+| `multus.image` | string | `multus` | Multus image name  |
+| `multus.repository` | string | `nfvpe` | Multus image repository  |
+| `multus.version` | string | `stable` | Multus image version  |
+| `multus.config` | string | `` | Multus CNI config if config is missing or empty then multus config will be automatically generated from the CNI configuration file of the master plugin (the first file in lexicographical order in cni-conf-dir)  |
+
+##### IPAM CNI Plugin Secondary Network
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `ipamPlugin.deploy` | bool | `true` | Deploy IPAM CNI Plugin Secondary Network  |
+| `ipamPlugin.image` | string | `whereabouts` | IPAM CNI Plugin image name  |
+| `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
+| `ipamPlugin.version` | string | `latest` | IPAM CNI Plugin image version  |
+
+### MacvlanNetwork Custom resource parameters
+
+| Name | Type | Default | description |
```

Comment:
Im kinda leaning towards not having this part of the chart.
but rather have the user define after deployment.

maybe the user needs more than one ? or none ?